src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/_factory.py

```diff
@@ -9,6 +9,6 @@ def iot_hub_service_factory(_):
     return get_mgmt_service_client(IotHubClient).iot_hub_resource
 
 def resource_service_factory(**_):
-    from azure.mgmt.resource.resources import ResourceManagementClient
+    from azure.mgmt.resource import ResourceManagementClient
```

Comment:
I thought this is a definite no-no with profile enabled CLI since it would always get latest (which may more may not be supported on the given cloud)?